libraries/AP_HAL_Linux/I2CDevice.cpp

```diff
@@ -166,6 +166,8 @@ bool I2CDevice::read_registers_multiple(uint8_t first_reg, uint8_t *recv,
 {
     const uint8_t max_times = I2C_RDRW_IOCTL_MAX_MSGS / 2;
 
+    first_reg |= _read_flag;
+
```

Comment:
this actually adds support for read flag in one additional method. The commit message looks actually a copy and paste from previous one.